src/client/play/js/camera.js

```diff
@@ -20,16 +20,28 @@ export default {
         this.y = -height / 2 + (config.firstTile.y + 2) * config.size + .85 * config.firstTile.x / 4 * config.size;
     },
 
+    zoomIn() {
+        this.targetZoom += .1;
+        this.updateZoomValue();
+    },
+
+    zoomOut() {
+        this.targetZoom -= .1;
+        this.updateZoomValue();
+    },
+
     /**
     * Animate camera zoom
     */
     zoom() {
         if (events.isZoomingOut()) {
-            this.targetZoom -= .1;
+            this.zoomIn();
```

Comment:
Why `zoomIn()` instead of `zoomOut()`? Previously the code was `targetZoom -= .1` which means now `zoomOut()`

Reply:
> Why `zoomIn()` instead of `zoomOut()`? Previously the code was `targetZoom -= .1` which means now `zoomOut()`

@borgeser Thanks for catching this! It was a bad case of copy/paste. :(

Reply:
Thank you both @rcjsuen @borgeser!